addons/web_tour/static/src/tour_service/tour_utils.js

```diff
@@ -30,10 +30,13 @@ export function callWithUnloadCheck(func, ...args) {
  * @returns {HTMLElement | null}
  */
 export function getScrollParent(element) {
-    if (!element) {
+    if (!element || element.nodeType === 9) {
```

Comment:
I don't know what `9` is. Use the named constant available on the `Node` object?

Reply:
> I don't know what `9` is. Use the named constant available on the `Node` object?

9 is for "#document" but it's no longer needed after the last change I made in this commit.